src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -320,6 +320,10 @@ HARDWARE_INTRINSIC(Sve2,          ShiftArithmeticRounded,
 HARDWARE_INTRINSIC(Sve2,          ShiftArithmeticRoundedSaturate,                                   -1,      -1,    {INS_sve_sqrshl,     INS_invalid,        INS_sve_sqrshl,     INS_invalid,        INS_sve_sqrshl,     INS_invalid,        INS_sve_sqrshl,     INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve2,          ShiftArithmeticSaturate,                                          -1,      -1,    {INS_sve_sqshl,      INS_invalid,        INS_sve_sqshl,      INS_invalid,        INS_sve_sqshl,      INS_invalid,        INS_sve_sqshl,      INS_invalid,        INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_LowMaskedOperation)
 HARDWARE_INTRINSIC(Sve2,          ShiftLeftAndInsert,                                               -1,      3,     {INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_invalid,        INS_invalid},     HW_Category_ShiftLeftByImmediate,  HW_Flag_Scalable|HW_Flag_HasImmediateOperand|HW_Flag_HasRMWSemantics)
+HARDWARE_INTRINSIC(Sve2,          ShiftLeftLogicalSaturate,                                         -1,      -1,    {INS_invalid,        INS_sve_uqshl,      INS_invalid,        INS_sve_uqshl,      INS_invalid,        INS_sve_uqshl,      INS_invalid,        INS_sve_uqshl,      INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_BaseTypeFromFirstArg|HW_Flag_EmbeddedMaskedOperation|HW_Flag_HasRMWSemantics)
+HARDWARE_INTRINSIC(Sve2,          ShiftLeftLogicalSaturateUnsigned,                                 -1,      -1,    {INS_invalid,        INS_sve_sqshlu,     INS_invalid,        INS_sve_sqshlu,     INS_invalid,        INS_sve_sqshlu,     INS_invalid,        INS_sve_sqshlu,     INS_invalid,        INS_invalid},     HW_Category_ShiftLeftByImmediate,  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_HasImmediateOperand|HW_Flag_HasRMWSemantics)
+HARDWARE_INTRINSIC(Sve2,          ShiftLeftLogicalWideningEven,                                     -1,      2,     {INS_invalid,        INS_invalid,        INS_sve_sshllb,     INS_sve_ushllb,     INS_sve_sshllb,     INS_sve_ushllb,     INS_sve_sshllb,     INS_sve_ushllb,     INS_invalid,        INS_invalid},     HW_Category_ShiftLeftByImmediate,  HW_Flag_Scalable|HW_Flag_HasImmediateOperand)
```

Comment:
also I think `ShiftLeftLogicalWideningEven` and `ShiftLeftLogicalWideningOdd` needs `HW_Flag_BaseTypeFromFirstArg` I guess?